AdobeIms/Test/Unit/Model/ConfigTest.php

```diff
@@ -36,28 +30,129 @@ class ConfigTest extends TestCase
     private $scopeConfigMock;
 
     /**
-     * Prepare test objects.
+     * @var UrlInterface|MockObject
+     */
+    private $urlMock;
+
+    /**
+     * Set up test mock objects
      */
     protected function setUp(): void
     {
-        $this->objectManager = new ObjectManager($this);
         $this->scopeConfigMock = $this->createMock(ScopeConfigInterface::class);
-        $this->config = $this->objectManager->getObject(
-            Config::class,
-            [
-                'scopeConfig' => $this->scopeConfigMock
-            ]
-        );
+        $this->urlMock = $this->createMock(UrlInterface::class);
+
+        $this->config = new Config($this->scopeConfigMock, $this->urlMock);
     }
 
     /**
-     * Get API key test.
+     * Test for \Magento\AdobeIms\Model\Config::getApiKey
      */
     public function testGetApiKey(): void
     {
         $this->scopeConfigMock->expects($this->once())
             ->method('getValue')
-            ->with(self::CONFIG_XML_PATH_API_KEY);
+            ->with(Config::XML_PATH_API_KEY);
+
         $this->config->getApiKey();
```

Comment:
Please check that correct value is returned instead of asserting the number of internal calls.

It could look like:
```
        $result = 'API KEY';
        $this->scopeConfigMock->method('getValue')
             ->with(Config::XML_PATH_API_KEY)
             ->willReturn($result);

         $this->assertSame($result, $this->config->getApiKey());
```